static/js/stream_list.js

```diff
@@ -84,7 +84,7 @@ exports.create_initial_sidebar_rows = function () {
     }
 };
 
-exports.build_stream_list = function () {
+exports.build_stream_list = function (flag) {
```

Comment:
If you want a boolean, pass it as a boolean (`true` or `false`) -- that's much more readable.

I would also expect we'd just pass an argument if needed, as `undefined` is falsey.